gb-ppu/src/drawing/pixel.rs

```diff
@@ -3,7 +3,7 @@ use crate::Color;
 use std::cell::Cell;
 use std::rc::Rc;
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
 pub struct Pixel {
     pub color: u8,
     pub palette: Rc<Cell<Palette>>,
```

Comment:
serializing a palette wouldn't cause problem ?
`serde` is able to link the palette correctly ?
I mean, we use `rc + refcell` here so we're sharing a palette, right ?